rbac is missing permissions.

when trying to use RBAC for k8s-coordinator extension theres this error raising from the coordinator:
```
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.20.0.1:443/apis/batch/v1/namespaces/analytics/jobs?labelSelector=druid.k8s.peons. Message: Forbidden!Configured service account doesn't have access. Service account may have been revoked. jobs.batch is forbidden: User "system:serviceaccount:analytics:analytics-nvq2-eks-operations-druid-coordinator" cannot list resource "jobs" in API group "batch" in the namespace "analytics".
```
this is due to missing permissions in the role configuration here: https://github.com/asdf2014/druid-helm/blob/aab58bf732c0ad64a58c38e79731a04d15e54b0a/charts/druid/templates/coordinator/role.yaml#L31

this rule block is missing:
```
rules:
  - apiGroups: ["batch"]
    resources: ["jobs"]
    verbs: ["get", "list", "watch"]
```
as described in the official docs.
https://druid.apache.org/docs/latest/development/extensions-contrib/k8s-jobs/#gotchas

i tried looking for a way to contribute and open a pr, but couldnt find a contribution guide.

--- a/charts/druid/Chart.yaml
+++ b/charts/druid/Chart.yaml
@@ -15,7 +15,7 @@
 
 apiVersion: v2
 name: druid
-version: 37.0.0
+version: 37.0.1
 appVersion: 37.0.0
 description: Apache Druid is a high performance real-time analytics database.
 keywords:

--- a/charts/druid/README.md
+++ b/charts/druid/README.md
@@ -28,7 +28,7 @@
 $ helm repo add druid-helm https://asdf2014.github.io/druid-helm/
 
 # Install chart
-$ helm install my-druid druid-helm/druid --version 37.0.0
+$ helm install my-druid druid-helm/druid --version 37.0.1
 ```
 
 

--- a/charts/druid/templates/coordinator/role.yaml
+++ b/charts/druid/templates/coordinator/role.yaml
@@ -34,6 +34,17 @@ rules:
     resources:
       - pods
       - configmaps
+      - pods/log
+    verbs:
+      - '*'
+  # Needed when the coordinator acts as overlord (druid.coordinator.asOverlord.enabled)
+  # with the druid-kubernetes-overlord-extensions task runner, which manages peon Jobs
+  # through the Kubernetes API. Mirrors the dedicated overlord role; see
+  # https://druid.apache.org/docs/latest/development/extensions-contrib/k8s-jobs/#gotchas
+  - apiGroups:
+      - batch
+    resources:
+      - jobs
     verbs:
       - '*'
 {{- end }}

